### PR TITLE
Pc/prerelease/project upgrade

### PR DESCRIPTION
**Pull request by pchittum**
Fixes for updated Flexipage metadata `itemInstances`. See Summer 20 release [notes](https://releasenotes.docs.salesforce.com/en-us/summer20/release-notes/rn_api_meta.htm). 

Also ran `force:project:update` fixing `sfdx-project.json` for proper attribute names.

### CODE REVIEW

File: sfdx-project.json [hunk 1]
```diff
@@ -9,10 +9,10 @@
         }
     ],
     "namespace": "",
-    "sourceApiVersion": "48.0",
+    "sourceApiVersion": "49.0",
     "sfdcLoginUrl": "https://login.salesforce.com",
     "packageAliases": {
-        "LWCRecipes": "0HoB0000000GmiJKAS",
+        "lWCRecipes": "0HoB0000000GmiJKAS",
```
Comment (kevinv11n) [inline sfdx-project.json:15]: Why the lower case L?

**Comment by muenzpraeger** [comment]
Wrong target branch

File: sfdx-project.json (hunk 1 above)
Comment (pchittum) [reply sfdx-project.json:15]: Ah that was from running `force:project:update`. I'll fix that.

**Comment by muenzpraeger** [comment]
I'd also push the `sfdx-lwc-jest` update, so that CI will be clean.

File: sfdx-project.json (hunk 1 above)
Comment (pchittum) [reply sfdx-project.json]: Updated with proper package name case.

**Pull request merged**